target/linux/ipq40xx/base-files/lib/upgrade/platform.sh

```diff
@@ -6,7 +6,8 @@ RAMFS_COPY_DATA='/etc/fw_env.config /var/lock/fw_printenv.lock'
 
 platform_check_image() {
 	case "$(board_name)" in
-	asus,rt-ac42u |\
```

Comment:
there shouldn't have been any change in the rt-ac42u line IMHO.

Reply:
As there hasn't been real change, I have to figure out how to get git to see that line is there as in orginal. Learning curve is steep. Manual diff edit or something?

Reply:
Maybe it's tabs vs spaces?